naga/tests/out/spv/wgsl-7995-unicode-idents.spvasm

```diff
@@ -0,0 +1,43 @@
+; SPIR-V
+; Version: 1.1
+; Generator: rspirv
+; Bound: 24
+OpCapability Shader
+OpExtension "SPV_KHR_storage_buffer_storage_class"
+%1 = OpExtInstImport "GLSL.std.450"
+OpMemoryModel Logical GLSL450
+OpEntryPoint GLCompute %19 "main"
+OpExecutionMode %19 LocalSize 1 1 1
+OpDecorate %4 NonWritable
+OpDecorate %4 DescriptorSet 0
+OpDecorate %4 Binding 0
+OpDecorate %5 Block
+OpMemberDecorate %5 0 Offset 0
+%2 = OpTypeVoid
+%3 = OpTypeFloat 32
+%5 = OpTypeStruct %3
+%6 = OpTypePointer StorageBuffer %5
+%4 = OpVariable  %6  StorageBuffer
+%9 = OpTypeFunction %3
+%10 = OpTypePointer StorageBuffer %3
+%12 = OpTypeInt 32 0
+%11 = OpConstant  %12  0
+%14 = OpConstant  %3  9001.0
```

Comment:
note: @jimblandy, @teoxoy: To my surprise, my Windows-generated snapshots apparently differed from CI's before I rebased onto latest `trunk`! My machine wants `9001.0` here, but CI wants `9001` with no decimal point. It's easy enough to fix and please CI, but…are y'all aware of this changing, for some reason?

Reply:
Might be related to what version of rspirv is in use (#7945).